test/cli/cli-reporter-no-success-assertsions.test.js

```diff
@@ -0,0 +1,24 @@
+describe('CLI reporter no success assertions', function () {
+    var noSuccessOutput = /✓\s+response\s+code\s+is\s+200/;
```

Comment:
The `✓` at the beginning of this pattern is causing Windows tests to fail :/. You'll have to remove it.

Reply:
Ahh. Yeah, will remove it. Still working on the cli.test.js.